lib/scan.ts

```diff
@@ -21,9 +21,15 @@ export async function scan(options: Options): Promise<ScanResult[]> {
       });
     }
     const artifacts: Artifact[] = [
-      { type: 'cpp-fingerprints', data: fingerprints },
+      { type: 'cpp-fingerprints', data: fingerprints, meta: {} },
+    ];
+    const scanResults: ScanResult[] = [
```

Comment:
I think that would be nice if we created few types on `lib/types.ts`
e.g. `type Cpp = 'cpp';` `type CppFingerPrints = 'cpp-fingerprints';` 
This way we can avoid string coupling here 💭